README.md

```diff
@@ -146,7 +146,7 @@ The core model is a shared BiLSTM with task-specific components. When a language
 
 The base model class is a MultiTaskModel. To add another model, first add the class of the model to modules/modules.py, and then add the model construction in ``make_sent_encoder()`` (called in build_model()) in src/models.py.
```

Comment:
Add something about how pooling and pairwise tasks are handled differently when using BERT?